### DISCUSSION TITLE
https://penguins-eggs.net/blog/triple-somersault

### DISCUSSION THREAD

**Discussion by giscus** [pieroproietti/penguins-blog#13] [General]
# https://penguins-eggs.net/blog/triple-somersault

Una volta creato e testato eggs per l'architettura ARM64, abbiamo il problema di testarlo su hardware reale.

https://penguins-eggs.net/blog/triple-somersault

**Comment by pieroproietti** [comment]
Will be nice to have some feedback...  thanks!
